version

```diff
@@ -1 +1 @@
-VERSION=0.1.257
+VERSION=0.1.258
```

Comment:
why do we need to update the version ?

Reply:
It is one of the checks we have for this repo, each PR should have a new version.